src/LaunchDarkly.EventSource/Resources.resx

```diff
@@ -117,31 +117,19 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="EventSourceService_Read_Timeout" xml:space="preserve">
-    <value>A timeout occurred while reading the request from the remote EventSource API</value>
+  <data name="ErrorReadTimeout" xml:space="preserve">
+    <value>Read timeout elapsed with no new data from server; connection may have been silently dropped</value>
```

Comment:
I added the note about how it might mean the connection was silently dropped because that might save people a little time in troubleshooting— that is, the main point of using a read timeout (and heartbeats) for streaming connections is that connections can be silently dropped. Someone who doesn't know that might assume that the timeout error must indicate a server problem.